packages/material-ui/src/Avatar/Avatar.js

```diff
@@ -70,7 +70,9 @@ function useLoaded({ src, srcSet }) {
     let active = true;
     const image = new Image();
     image.src = src;
-    image.srcSet = srcSet;
+    if (srcSet) {
```

Comment:
We don't need this check since we already checked for truthy-ness earlier: https://github.com/mui-org/material-ui/pull/23286/files#diff-1b9570feecbd06a92b6ff9e5d7397dc11d978bcbe22584ff4cdb9990f91529b1L64-L66

Reply:
We need it because that only checks if both are truthy-ness values, which means that if `src` is a truthy-ness value, but `srcSet` is not, at line 73 the `srcSet` can be `undefined`.

https://repl.it/@matheuspiment/fixavatar-srcset